hebrewreader.py

```diff
@@ -159,23 +171,30 @@ def generate(passages, include_voca, combine_voca, clearpage_before_voca,
     voca = set()
 
     for passage_text in passages:
-        passage = parse_passage(passage_text)
+        passage = parse_passage(passage_text, lang)
 
         passage_pretty = '{} {}:{} - {}:{}'.format(
             passage['book'].replace('_', ' '),
             passage['startchap'], passage['startverse'],
             passage['endchap'], passage['endverse'])
         tex.write(r'\def\thepassage{%s}' % passage_pretty)
 
-        try:
-            api = load_data(passage)
-            text, words = get_passage_and_words(passage, api)
-        except:
-            raise ValueError('Could not find reference "{}"'.format(passage_text))
+        api = load_data(passage, lang)
+        text, words = get_passage_and_words(passage, api, lang)
 
-        tex.write('\n\n' + templates['pretext'])
+        if lang == "hebrew":
+            tex.write('\n\n' + templates['pretext'])
+        elif lang == "syriac":
+            tex.write('\n\n' + templates['pretext_syr'])
+        elif lang == "greek":
+            tex.write('\n\n' + templates['pretext_grk'])
         tex.write('\n'.join(text))
-        tex.write('\n' + templates['posttext'])
+        if lang == "hebrew":
+            tex.write('\n' + templates['posttext'])
+        elif lang == "syriac":
+            tex.write('\n' + templates['posttext_syr'])
+        elif lang == "greek":
+            tex.write('\n' + templates['posttext_grk'])
```

Comment:
If you rename the files to `pretext_LANGUAGE.tex`, you can simply use `templates['pretext_' + lang]` here.